demo/mei_dj_free_demo.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !pip3 install git+https://github.com/sinzlab/nnvision.git@model_builder
</pre><p>pip install -q ... to make it install quietly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/mei/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>